package.json

```
@@ -1,14 +1,12 @@
 {
-  "name": "safe-token",
+  "name": "@safe-global/safe-token",
   "version": "1.0.0",
   "description": "Repository for Safe token related contracts",
   "homepage": "https://www.safe.global",
```

```suggestion
  "homepage": "https://safe.global",
```

We don't use `www` anymore right?